packages/fuel-indexer-schema/src/lib.rs

```diff
@@ -154,94 +154,178 @@ impl FtColumn {
     pub fn query_fragment(&self) -> String {
         match self {
             FtColumn::ID(value) => {
-                format!("{}", value)
+                if let Some(val) = value {
```

Comment:
Maybe we can cleanup all of this `if let Some(x) = y { a } else { b }`?

`String::from("NULL")` looks ripe for a default value `const NULL_VALUE: &str = "NULL" (maybe using `lazy_static for this)